src/nni_manager/training_service/test/localTrainingService.test.ts

```diff
@@ -72,6 +74,38 @@ describe('Unit Test for LocalTrainingService', () => {
         chai.expect(jobDetail.status).to.be.equals('USER_CANCELED');
     }).timeout(20000);
 
+    it('Get trial log', async () => {
+        // set meta data
+        const trialConfig: string = `{\"command\":\"python3 mockedTrial.py\", \"codeDir\":\"${localCodeDir}\",\"gpuNum\":0}`
```

Comment:
I am wondering whether this `mockedTrial.py` will cause some conflicts regarding log content.  It may generate log which make the log file content is not as expected.

Reply:
How about change the command to `sleep 5` ?

Reply:
fixed. now use the original trialConifig in line 23